ulfhara.lic

```diff
@@ -122,6 +123,23 @@ class UlfHara
         combo
       when /help any of our wounded you encounter/
         fput("accept task")
+        if DRStats.empath? && !DRSpells.active_spells.include?('Regenerate') && @settings.waggle_sets['healing']['Regenerate']
+          data = @settings.waggle_sets['healing']['Regenerate']
+          check_discern(data, @settings) if data['use_auto_mana']
```

Comment:
should we assign data to the return from this? Since `check_discern` returns the modified data

Reply:
the only reason to assign the return is if you haven't don't have is set already. Which I did just above.

Reply:
changed.